README.md

```diff
@@ -318,3 +318,7 @@ The Wikisource logo is included as `public/img/Wikisource-logo.svg`,
 as an optimized form of the logo
 found at [commons.wikimedia.org/wiki/File:Wikisource-logo.svg](https://commons.wikimedia.org/wiki/File:Wikisource-logo.svg)
 and subject to the licence restrictions specified on that page.
+
+The PNG version in `resources/Wikisource-logo.svg.png`
+is [the Commons derivative](https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/160px-Wikisource-logo.svg.png)
```

Comment:
The word "derivative" here suggests visual modifications. How about something like this?

```suggestion
is [the Commons auto-generated 160px raster version](https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Wikisource-logo.svg/160px-Wikisource-logo.svg.png)
```

Reply:
Sounds good! I'm not sure I get the same idea from 'derivative', but your wording is certainly clearer.